libs/api-client-bear/src/execution/experimental-executions.ts

```diff
@@ -70,30 +68,30 @@ const getBaseMetricTitle = partial(getMetricTitle, "");
 
 const CONTRIBUTION_METRIC_FORMAT = "#,##0.00%";
 
-function getPoPDefinition(measure: IMeasure) {
-    return get(measure, ["definition", "popMeasureDefinition"], {});
+function getPoPDefinition(measure: any) {
+    return measure?.definition?.popMeasureDefinition ?? {};
```

Comment:
I assume we don't have better typings/type guards for these methods?

Reply:
We don't unfortunately :-/